shells/diagnostic/user-context/index.js

```diff
@@ -0,0 +1,67 @@
+import '../../lib/database/firebase-support.js';
+import '../../lib/platform/loglevel-web.js';
+import {Utils} from '../../lib/runtime/utils.js';
+import {SyntheticStores} from '../../lib/runtime/synthetic-stores.js';
+import {StoreObserver} from '../../lib/components/context/store-observer.js';
+import {ArcHandleListener, ArcMetaListener, ProfileListener, ShareListener} from '../../lib/components/context/context-listeners.js';
+import {ProfileDisplayMixin, ShareDisplayMixin, ArcMetaDisplayMixin, ArcHandleDisplayMixin} from './table-mixins.js';
+import '../../../modalities/dom/components/arc-tools/store-explorer.js';
+
+const storage = `firebase://arcs-storage.firebaseio.com/AIzaSyBme42moeI-2k8WgXh-6YK_wYyjEXo4Oz8/0_7_0/sjmiles`;
```

Comment:
is `sjmiles` here on purpose?

Reply:
Yes. This is a publicKey that stores all my stuff. I put my ldap there just for convenience, there is nothing special about it.

Reply:
A user in modern times is a `publicKey`, not just a simple name. I gave a micro-demo of this concept, but I think you were OOO.